packages/sdk/docs/mpl-token-auth-rules/classes/AdditionalSignerCheckFailedError.html

```diff
@@ -25,7 +25,7 @@ <h4>Hierarchy</h4>
 <ul class="tsd-hierarchy">
 <li><span class="target">AdditionalSignerCheckFailedError</span></li></ul></li></ul></section><aside class="tsd-sources">
 <ul>
-<li>Defined in <a href="https://github.com/metaplex-foundation/token-authorization-rules/blob/92b1330/packages/sdk/src/generated/errors/index.ts#L120">packages/sdk/src/generated/errors/index.ts:120</a></li></ul></aside>
+<li>Defined in <a href="https://github.com/metaplex-foundation/token-authorization-rules/blob/7493263/packages/sdk/src/generated/errors/index.ts#L120">work/authenticatooor/packages/sdk/src/generated/errors/index.ts:120</a></li></ul></aside>
```

Comment:
Nit: old repo name.  It probably redirects but could you regenerate with latest repo name?